cmd/api/api_server.go

```diff
@@ -61,18 +85,35 @@ func getClusters(c *gin.Context) {
 		}
 	}
 
-	c.PureJSON(http.StatusOK, clusters)
+	response := ClusterListResponse{
+		Count:    len(clusters),
+		Clusters: clusters,
```

Comment:
We return `null` for an empty response
Does it make sense to return an empty array instead?
Like
```
{"count":0,"clusters":[]}
```

Reply:
It's already returning that. That's one of the purposes of including a new structure.
![sc](https://github.com/RHEcosystemAppEng/cluster-iq/assets/51123977/7b1681d7-3dcc-4e2b-bb5e-5a8a6f64849d)

Reply:
Nice! makes a lot of sense. Will include it